Makefile

```diff
@@ -655,9 +655,8 @@ kind-image-fast-agent: kind-ready build-cli build-agent ## Build cilium cli and
 			docker exec -ti $${node_name} rm -f "${dst}/cilium-agent"; \
 			docker cp "./daemon/cilium-agent" $${node_name}:"${dst}"; \
 			docker exec -ti $${node_name} chmod +x "${dst}/cilium-agent"; \
-			\
-			kubectl --context=kind-$${cluster_name} delete pods -n kube-system -l k8s-app=cilium --force; \
 		done; \
+		kubectl --context=kind-$${cluster_name} rollout restart -n kube-system daemonset -l app.kubernetes.io/name=cilium-agent; \
```

Comment:
I agree with the node vs cluster deletion but I'm not convinced with the "rollout restart". The reason behind delete pods is that we want to deploy Cilium as fast as possible. The rollout sounds like nice way to deploy the new Cilium version but it's slower.

Reply:
Yeah, fair enough. Unless we patch the DaemonSet to have a larger surge this will add time. I'll set it back.

Reply:
Fixed.